processing/src/main/java/org/apache/druid/query/rowsandcols/DefaultGroupPartitioner.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.rowsandcols;
+
+import org.apache.druid.java.util.common.ISE;
+import org.apache.druid.query.rowsandcols.column.Column;
+import org.apache.druid.query.rowsandcols.column.ColumnAccessor;
+
+import java.util.List;
+
+@SuppressWarnings("unused")
+public class DefaultGroupPartitioner implements GroupPartitioner
```

Comment:
Yes, with the one exception that this one isn't used anymore.  It's really just sitting around in the code for posterity rather than any specific usage.  Which probably means it's worth deleting.  I'd prefer to do that in a subsequent PR though.